tests/integration/dynamodb2/test_highlevel.py

```diff
@@ -716,3 +716,68 @@ def test_query_after_describe_with_gsi(self):
 
         for rs_item in rs:
             self.assertEqual(rs_item['username'], ['johndoe'])
+
+    def test_update_table_online_indexing_support(self):
```

Comment:
@xethorn, these tests are run against the live DynamoDB service using your configured credentials. It's possible that you could run against a DynamoDB Local service, but it is not used by default.